### PR TITLE
Story/epgr 12

### PR DESCRIPTION
**Pull request by Mihika0409**
Graph on home page should scroll down as user scrolls through the list of people

### CODE REVIEW

File: grazer/src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -10,105 +10,120 @@
 $(document).ready(function() {
```
Comment (jdamerow) [inline grazer/src/main/webapp/WEB-INF/views/home.jsp]: lets keep the 4 spaces indentation

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
ok to test

File: grazer/src/main/webapp/WEB-INF/views/home.jsp [hunk 2]
```diff
@@ -10,105 +10,120 @@
...
+                        marginTop: $(window).scrollTop() - offset.top + topPadding
+                    });
+                } else {
+                    $("#graph").stop().animate({
+                        marginTop: 0
+                    });
+                };
+            });
+        });
+        window.onresize = function(){ location.reload(); }
```
Comment (jdamerow) [inline grazer/src/main/webapp/WEB-INF/views/home.jsp]: does this reload the page as soon as the user resizes the window? the user would loose on whatever part he zoomed in on. that would be pretty annoying.

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: grazer/src/main/webapp/WEB-INF/views/home.jsp [hunk 3]
```diff
@@ -90,23 +90,14 @@ $(document).ready(function() {
...
-                        marginTop: 0
-                    });
-                };
-            });
-        });
-</script>
+
+<style>
+    .graph-outer { position: relative; }
+
```
Comment (chiraag-subramanian) [inline grazer/src/main/webapp/WEB-INF/views/home.jsp]: Do we need the style position:relative for .graph-outer. The position:relative tag is accompanied by either of these offset : top, bottom, right or left
https://www.w3schools.com/css/css_positioning.asp

**Review by abhikrish503501 — APPROVED** [review]

**Review by chiraag-subramanian — APPROVED** [review]

**Comment by jdamerow** [comment]
formatting is still not taken care of

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
Instead of the graph being fixed, can we please simply make the name list scrollable (every stays, just the names scroll)? The way it is right now looks funny because there is a lot of white space above the graph.

**Comment by jdamerow** [comment]
ok to test

**Review by jdamerow — APPROVED** [review]

**Pull request merged**